pkg/remote/remote.go

```diff
@@ -223,13 +249,18 @@ type GoGetter struct {
 func (g *GoGetter) Get(wd, src, dst string) error {
 	ctx := context.Background()
 
+	opts := []getter.ClientOption{}
+
+	// enable insecure features
+	opts = append(opts, getter.WithInsecure())
+
 	get := &getter.Client{
 		Ctx:     ctx,
 		Src:     src,
 		Dst:     dst,
 		Pwd:     wd,
-		Mode:    getter.ClientModeDir,
-		Options: []getter.ClientOption{},
+		Mode:    getter.ClientModeAny,
```

Comment:
This changes the meaning of this `func (g *GoGetter) Get(wd, src, dst string) error`, right?
Previously, `dst` had been a directory. Since this change though, `dst` can either be a file or directory, and the caller of the function needs to handle both cases gracefully?